MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+
+* To find the item appears in the view.
+* To customize the appearing item to change the background color using converter.
+
+### ItemDisappearing
+
+The [ItemDisappearing](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemDisappearing) event is raised when the items disappearing in the view on scrolling, disposing, and navigating from one page to another page. The [ItemDisappearingEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDisappearingEventArgs.html) has the following member which provides the information on disappearing Items:
+
+* [ItemData](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDisappearingEventArgs.html#Syncfusion_Maui_ListView_ItemDisappearingEventArgs_ItemData): The underlying data associated with the disappearing item.
+
+{% tabs %}
+{% highlight c# %}
+
+listView.ItemDisappearing += listView_ItemDisappearing;
+
+private void listView_ItemDisappearing(object sender, Syncfusion.Maui.ListView.ItemDisappearingEventArgs e)
+{
+    if (e.ItemData == viewModel.BookInfo[0])
+        Debug.WriteLine((e.ItemData as BookInfo).BookName);
+
+    // If the ItemData value is "Footer" then it's a Footer item.
+    if (e.ItemData == "Footer")
+        Debug.WriteLine("Footer is Disappeared");
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The [ItemDisappearing](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemDisappearing) event used for the following use cases:
+
+* To find the item disappears in the view.
+
+## Loading data from SQLite online database
+
+The .NET MAUI (SfListView) allows binding the data from online database with the help of `Azure App Service`. To perform this, follow the steps:
+
+Step 1: Get URL to store the data online.
+Step 2: Create table using `AppServiceHelpers`.
+Step 3: Populate the data into the table using the ObservableCollection `Items`.
+Step 4: Bind it to SfListView using [SfListView.ItemSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemsSource) property.
+Step 5: Refer to the following link to know more about working with `Azure App Service`.
+        https://blog.xamarin.com/add-a-backend-to-your-app-in-10-minutes/ 
+
+N> Install the following NuGet packages to successfully run the application:
+
+* Microsoft.Azure.Mobile.Client(v.2.1.1)
+* Microsoft.Azure.Mobile.Client.SQLiteStore(v.2.1.1)
+* AppService.Helpers (Does not support UWP platform)
+* AppService.Helpers.Forms (Does not support UWP platform)
+
+Refer to the following code which illustrates, how to initialize the library with the URL of the `Azure` Mobile App and registering the Model with the client to create a table.

```

Comment:
Refer to the following code to see how to initialize the library with the URL of the `Azure` Mobile App and register the Model with the client to create a table.

Reply:
I have modified the line as per the concerns provided